less/plugins/adapt-contrib-boxmenu/boxMenu.less

```diff
@@ -43,4 +43,8 @@
   &__instruction a {
     .link-text;
   }
+
+  &__item-container {
+    margin-top: 1.5rem;
```

Comment:
Could this be set up as a variable and defined here perhaps? https://github.com/adaptlearning/adapt-contrib-vanilla/blob/master/less/_defaults/_spacing-containers.less#L13